webapp/src/components/Navbar.tsx

```diff
@@ -1,21 +1,34 @@
 import React, { useContext } from 'react';
 import { Link } from 'react-router-dom';
 import { Button, Grid, IconButton } from '@mui/material';
+import styled from '@emotion/styled';
 import CalendarMonthIcon from '@mui/icons-material/CalendarMonth';
+import HomeIcon from '@mui/icons-material/Home';
 import PeopleIcon from '@mui/icons-material/People';
 
 import MeetingsDrawerContext from './MeetingsDrawerContext';
 
+const StyledLogo = styled.ul`
+  &:hover {
+    color: blue;
+  }
+`;
+
 const Navbar = () => {
   const { open: openMeetingsDrawer } = useContext(MeetingsDrawerContext);
   return (
     <Grid alignItems="center" container px={3}>
       <Grid item xs>
         <Link to="/" style={{ textDecoration: 'none', color: 'black' }}>
-          <h1>Rhizone</h1>
+          <StyledLogo>
+            <h1>Rhizone</h1>
+          </StyledLogo>
```

Comment:
I noticed that the location of the 'Rhizone' was moved a little to the right. I think it looks fine but just wondering if the rest of the team wants to change the location of the logo?

Reply:
That is a great question! Let's talk about it in the next stand up. Thank you for noticing it!